youtube_transcript_api/test/test_formatters.py

```diff
@@ -0,0 +1,99 @@
+from unittest import TestCase
+from mock import MagicMock
+import json
+
+from youtube_transcript_api.formatters import (
+    JSONTranscriptFormatter,
+    parse_timecode,
+    SRTTranscriptFormatter,
+    TextTranscriptFormatter,
+    TranscriptFormatter,
+    TranscriptFormatterFactory
+)
+
+
+class TestTranscriptFormatters(TestCase):
```

Comment:
I'd prefer having individual test cases for each formatters. I think this would make it cleaner and more extensible, when other formatters are added in the future. You could have then share the same base class if you want them to use the same setup method.